src/ekat/ekat_parameter_list.hpp

```diff
@@ -64,7 +64,7 @@ class ParameterList {
 
   // Check methods, to determine the type of a node
   template<typename T>
-  bool isType (const std::string& name);
+  const bool isType (const std::string& name);
```

Comment:
Sorry, this is not what I meant with marking the method const. This means that the returned bool is constant (and a decent compiler will actually issue a warning here, since you're returning by value, so const is pointless). To mark the method const, you need to put `const` _after_ the signature, right before the semicolon:

```
bool isType (...) const;
```

Reply:
Ahhh, my bad.  I think it is fixed now.